### PR TITLE
YOTJ 6 cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/silent_extraction.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Name:Silent Extraction
+ManaCost:G U
+Types:Instant
+A:SP$ Pump | TargetType$ Spell | TgtZone$ Stack | ValidTgts$ Card | SubAbility$ DBCounter | SpellDescription$ Choose target spell.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/silent_extraction.txt:4]: why extra Pump?

File: forge-gui/res/cardsfolder/upcoming/switchgrass_grazer.txt [hunk 2]
```diff
@@ -0,0 +1,13 @@
+Name:Switchgrass Grazer
+ManaCost:2 R R
+Types:Creature Horse Mount
+PT:4/4
+K:Haste
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigDamage | TriggerDescription$ Whenever CARDNAME attacks, it deals 1 damage to any target. If CARDNAME is saddled and a creature was dealt damage this way, that creature perpetually gains "This creature can't block" and "Damage isn't removed from this creature during cleanup steps."
+SVar:TrigDamage:DB$ DealDamage | ValidTgts$ Any | NumDmg$ 1 | RememberDamaged$ True | SubAbility$ DBAnimate
+SVar:DBAnimate:DB$ Animate | Defined$ Remembered | staticAbilities$ CantBlock,NoDamageRemoval | Duration$ Perpetual | ConditionDefined$ Remembered | ConditionPresent$ Creature | ConditionPresent2$ Card.Self+IsSaddled | Types$ Angel | SubAbility$ DBCleanup
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/switchgrass_grazer.txt:8]: Angel?

File: forge-gui/res/cardsfolder/upcoming/silent_extraction.txt (hunk 1 above)
Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/silent_extraction.txt:4]: I think it's need to allow you to seek if the card isn't countered ?

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/silent_extraction.txt:4]: doesn't make sense
Targeted stays the same either way

**Review by tool4ever — APPROVED** [review]

**Pull request merged**